src/lineCompare.component.ts

```diff
@@ -5,24 +5,64 @@ import { DiffMatchPatchService } from './diffMatchPatch.service';
 @Component({
   selector: 'dmp-line-compare',
   styles: [`
+    div.dmp-line-compare {
+      display: flex;
+      flex-direction: row;
+      border: 1px solid #808080;
+      font-family: Consolas, Courier, monospace;
+      width: 911px;
```

Comment:
in my case (Angular 5) I had to override this width attribute in a global css file to width: auto; in order to have the div not exceed the boundaries of a [ngx-bootstrap modal](https://valor-software.com/ngx-bootstrap/#/modals) I am putting it in.